sdk/storage/azure-storage-blobs/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0-beta.1
```

Comment:
Get rid of this file and use a version.h file in the `azure/storage/blobs/version.h`.  See azure-core folder for the contents of that file.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/version.txt

Reply:
azure core folder also has this file.

Reply:
I think we should save the version information only at one place, either in version.txt or in version.hpp. I'm fine either way.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/543 I created an issue to track that. This is not a big deal. I think we can figure it out in the future.